.github/workflows/lint.yml

```diff
@@ -31,3 +31,19 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           version: latest
           args: --check .
+
+  format-queries:
+    name: Lint queries
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Prepare
+        run: |
+          wget https://github.com/neovim/neovim/releases/download/nightly/nvim-linux64.tar.gz
```

Comment:
Is it intended to use nightly for this :thinking:

Reply:
Yes, why not? (I've adjusted so the script works on 0.9.x, but that will give us some early warning if things break.)